orchestrator/index.js

```diff
@@ -0,0 +1,63 @@
+const request = require('request-promise')
```

Comment:
Actually, the linter doesn't catch the errors, because you out it in the root folder :) I was wondering why not. I think the server folder is a better place for this.
```
server/
   orchestrator/
        orchestrator.js <- put the content here
        index.js <- see other modules, basically reexport
```